cix.sh

```diff
@@ -1,56 +1,6 @@
 #!/bin/bash
 set -euo pipefail
 
-plugins_min_versions_path=core/src/main/resources/plugins_min_versions.txt
```

Comment:
I added a commit to your branch, this variable is needed by the new script where you extracted the code.

Reply:
thx